src/main/resources/application.yml

```diff
@@ -72,6 +72,60 @@ spring:
     user: admin
     password: secret
 
+survey-svc:
+  request-classifier-types-list-path: /surveys/{surveyId}/classifiertypeselectors
+  request-classifier-types-path: /surveys/{surveyId}/classifiertypeselectors/{selectorId}
+  survey-detail-path: /surveys/{surveyId}
+  survey-ref-path: /surveys/ref/{surveyRef}
+  connection-config:
+    scheme: http
+    host: localhost
+    port: 8080
+    username: admin
+    password: secret
+    # how many times should we attempt connection on failure
+    retry-attempts: 5
+    # sleep between retries
+    retry-pause-milli-seconds: 5000
+    # time to estab connection
+    connect-timeout-milli-seconds: 5000
+    # time to read response
+    read-timeout-milli-seconds: 5000
+
+
+collection-instrument-svc:
+  request-collection-instruments: /collection-instrument-api/1.0.2/collectioninstrument
+  request-collection-instruments-count: /collection-instrument-api/1.0.2/collectioninstrument/count
+  connection-config:
+    scheme: http
+    host: localhost
+    port: 8002
+    username: admin
+    password: secret
+    retry-attempts: 5
+    retry-pause-milli-seconds: 5000
+    connect-timeout-milli-seconds: 5000
+    read-timeout-milli-seconds: 5000
+
+
+party-svc:
+  request-party-path: /party-api/v1/parties/type/{sampleUnitType}/ref/{sampleUnitRef}
```

Comment:
Will the sampleUnitType always be B?  The card I'm working on removes this endpoint.  Would it be possible to change it to `/businesses/{sampleUnitRef}` (or something like that.. can't remember off the top of my head).  Though if we're trying to make this simpler, which isn't a bad idea, it's not a big deal to copy/paste it now (as we know it works) then change it later.

Reply:
Yeah we can do that. Is it ok to do it as a separate PR though, as this bit is basically copied from collection exercise so might go to have traceability of where it came from